pkgs/development/compilers/llvm/common/lldb.nix

```diff
@@ -154,6 +154,13 @@ stdenv.mkDerivation (rec {
     ln -s $out/bin/*-vscode $out/share/vscode/extensions/llvm-org.lldb-vscode-0.1.0/bin
   '';
 
+  # TODO: Hack to make dynamic linking work again. Find a better way to do this using the build-system machinery
+  postFixup = if (stdenv.isLinux && lib.versionAtLeast release_version "16") then ''
```

Comment:
I guess this is the biggest problem—such hacky workarounds have a tendency to stick which is annoying. Can you tell what changed so this broke between lldb 15 and lldb 16?

Reply:
As I noted in the PR description - I can't figure it out: whole llvm build procedure is quite... enigmatic. I understand the gravity of the hack. My reasoning is: if there is no one to figure it out in a reasonable timeframe it should be accepted as is unless there are potential issues which are unrelated and may result in fallout far bigger than completely broken LLDB.